### PR TITLE
Add nix package definition and buildkite configuration

### PR DESCRIPTION
**Pull request by Lucus16**

### CODE REVIEW

File: default.nix [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{ pkgs ? import ./pkgs.nix }:
+
+with pkgs;
+
+let
+  hex = fetchurl {
```
Comment (Lucus16) [inline default.nix]: Maybe we should add `hex` to `serokell-closure`?

File: default.nix [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    url = "https://repo.hex.pm/installs/1.6.0/hex-0.18.1.ez";
+    sha512 = "9c806664a3341930df4528be38b0da216a31994830c4437246555fe3027c047e073415bcb1b6557a28549e12b0c986142f5e3485825a66033f67302e87520119";
+  };
+in
+
+(callPackage (mixToNix ./.) {}).overrideAttrs (oldAttrs: rec {
+  name = "educator_aaa";
+  version = "0.0.1";
```
Comment (Lucus16) [inline default.nix:11]: `version` is also defined in `mix.exs` and needs to match. Can I extract it using nix or should I just extract it with a bash command in the `installPhase`?

File: default.nix (hunk 1 above)
Comment (mkaito) [reply default.nix]: Probably a good idea, yes. Can you do that?

Comment (mkaito) [reply default.nix]: Actually, why not submit upstream to nixpkgs?

File: default.nix [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+with pkgs;
+
+let
+  hex = fetchurl {
+    url = "https://repo.hex.pm/installs/1.6.0/hex-0.18.1.ez";
+    sha512 = "9c806664a3341930df4528be38b0da216a31994830c4437246555fe3027c047e073415bcb1b6557a28549e12b0c986142f5e3485825a66033f67302e87520119";
+  };
+in
+
+(callPackage (mixToNix ./.) {}).overrideAttrs (oldAttrs: rec {
```
Comment (mkaito) [inline default.nix:9]: This seems oddly wrong. Why are we overriding?

Comment (Lucus16) [reply default.nix:9]: `serokell/mix-to-nix` wasn't designed to work with `distillery` releases. They actually seem pretty nice, so maybe we should switch `mix-to-nix` and `hermetic` over to them as well. But this works for now.

**Pull request merged**